tests/components/modbus/test_modbus_sensor.py

```diff
@@ -0,0 +1,337 @@
+"""The tests for the Modbus sensor component."""
+import unittest
+from unittest import mock
+
+from homeassistant.const import CONF_NAME, CONF_OFFSET, CONF_PLATFORM
+from homeassistant.components.modbus import DEFAULT_HUB, DOMAIN as MODBUS_DOMAIN
+from homeassistant.components.modbus.sensor import (
+    CONF_COUNT,
+    CONF_DATA_TYPE,
+    CONF_PRECISION,
+    CONF_REGISTER,
+    CONF_REGISTER_TYPE,
+    CONF_REGISTERS,
+    CONF_REVERSE_ORDER,
+    CONF_SCALE,
+    DATA_TYPE_CUSTOM,
+    DATA_TYPE_FLOAT,
+    DATA_TYPE_INT,
+    DATA_TYPE_UINT,
+    REGISTER_TYPE_HOLDING,
+    REGISTER_TYPE_INPUT,
+    PLATFORM_SCHEMA,
+    setup_platform,
+)
+from tests.common import get_test_home_assistant
+
+
+class TestModbusRegisterSensor(unittest.TestCase):
```

Comment:
New tests must be standalone pytest test functions.

Reply:
Done